src/components/ResultViewGrid.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from "react";
+import { Modal, Button, CardActionArea } from "@mui/material";
+
+import type { DisplayResult } from "./ResultView";
+import ResultViewList from "./ResultViewList";
+
+import Card from "@mui/material/Card";
+import CardActions from "@mui/material/CardActions";
+import CardContent from "@mui/material/CardContent";
+import CardMedia from "@mui/material/CardMedia";
+
+const ResultViewGrid = ({
+  displayResult,
+}: {
+  displayResult: DisplayResult;
+}) => {
+  const { displayName, displayOrganisation, displayRole, thumbnailImageUrl } =
+    displayResult;
+
+  const [isModalOpen, setIsModalOpen] = useState(false);
+
+  return (
+    <Card
+      className="sui-result"
+      style={{
+        maxWidth: "220px",
+        padding: "0px",
+      }}
+    >
+      <CardActionArea
+        onClick={() => setIsModalOpen(true)}
+        style={{
+          display: "flex",
+          flexFlow: "column nowrap",
+          alignItems: "stretch",
+          justifyContent: "flex-start",
+          height: "100%",
+        }}
+      >
+        <CardMedia
+          component="img"
+          image={thumbnailImageUrl}
+          style={{ width: "100%" }}
+        />
+        <CardContent>
+          {displayName && (
+            <span
+              className="sui-result__title"
+              style={{ fontSize: 18 }}
+              dangerouslySetInnerHTML={{ __html: displayName }}
+            />
+          )}
+          <ul className="sui-result__details" style={{ padding: 0 }}>
+            {displayOrganisation && (
+              <li>
+                <span
+                  className="sui-result__value"
+                  style={{ fontSize: 14 }}
+                  dangerouslySetInnerHTML={{ __html: displayOrganisation }}
+                />
+              </li>
+            )}
+            {displayRole && (
+              <li>
+                <span
+                  className="sui-result__value"
+                  style={{ fontSize: 14 }}
+                  dangerouslySetInnerHTML={{ __html: displayRole }}
+                />
+              </li>
+            )}
+          </ul>
+        </CardContent>
+        <CardActions
+          style={{ display: "flex", flexGrow: 1, alignItems: "flex-end" }}
+        >
+          <Button onClick={() => setIsModalOpen(true)} style={{ fontSize: 12 }}>
+            Read More
+          </Button>
+        </CardActions>
+      </CardActionArea>
+      <Modal
+        className="sui-result__modal"
+        open={isModalOpen}
+        onClose={() => setIsModalOpen(false)}
+        aria-labelledby="modal-title"
+        aria-describedby="modal-description"
+        style={{
+          display: "flex",
+          alignItems: "flex-start",
+          margin: "auto",
+          maxWidth: "800px",
+          maxHeight: "80%",
+          overflow: "scroll",
```

Comment:
Should change this from scroll to auto, such that unnecessary scroll bars don't show up everywhere

Sample Test

![image](https://user-images.githubusercontent.com/18213722/170832794-40b78c6d-a623-4303-9b10-9a5e7f5e182e.png)

![image](https://user-images.githubusercontent.com/18213722/170832774-36fd1da8-e599-4f3a-b13b-0157d63d35d7.png)